src/components/Garden/Feed/ProposalInfo.tsx

```diff
@@ -31,12 +37,10 @@ function ProposalInfo({
     requestToken.decimals
   )
 
-  const hideRequestedInfo = Number(proposal.requestedAmount) === 0
-
   return (
     <div onClick={onSelectProposal}>
       <ProposalDescription proposal={proposal} />
-      {proposal.type !== ProposalTypes.Decision && !hideRequestedInfo && (
+      {proposal.type === ProposalTypes.Proposal && (
```

Comment:
why we changed this here ? before it meant to be for proposal types of proposals and suggestions

Reply:
If I'm not mistaken because the `hideRequestedInfo` condition we only display for `Proposal` type. So I change to be more explicit.